src/Microsoft.OData.Core/PublicAPI/netcoreapp3.1/PublicAPI.Unshipped.txt

```diff
@@ -1 +0,0 @@
-Microsoft.OData.UriParser.ODataPath.this[int index].get -> Microsoft.OData.UriParser.ODataPathSegment

```

Comment:
why do we have such?

Reply:
When doing a release, we copy the changes in the `*.Unshipped.txt` file to `*.Shipped.txt` file.
We do so by running the `.\tools\IncrementVersion.cmd`. See [here](https://github.com/OData/odata.net/blob/master/docs/release.md#initiate-the-release)